dockerfiles/google/Dockerfile

```diff
@@ -52,6 +54,9 @@ RUN apk --no-cache add \
         g++ \
         make \
         ruby-dev && \
+    # Following is needed for various native exptension issues on Alpine with protobuf and grpc plugins
```

Comment:
This is a little vague, so I might link to the most relevant issue(s) here, or back to the description of this PR.

Reply:
Thanks. So there are like 4 different issues I've hit and each of them requires some part of the fix and if not present will lead to different issue(s):
- doing the `gem update`
- explicitly installing `google-protobuf` with `ruby` platfotm
- same for `grpc` and it's dependency `etc`
- pinning down the protobuf version at `3.6.1`

I think thiat is a bit too long to explain in the code. All of the issue(s) are referenced in the commit message (as well as in the PR description), so I'll remove the comment here.